.travis.yml

```diff
@@ -30,7 +30,8 @@ before_script:
   # nltk 3.2.2 dropped support for Python 3.3
   - >
     if [[ "$TRAVIS_PYTHON_VERSION" != "3.3" ]]; then
-      python -m nltk.downloader punkt maxent_treebank_pos_tagger averaged_perceptron_tagger
+      python -m nltk.downloader \
+      punkt maxent_treebank_pos_tagger averaged_perceptron_tagger
```

Comment:
I would indent by another 2 spaces so someone can immediately see that those two lines have a relationship ;)

Reply:
@Makman2 you mean another two whitespaces before 'punkt' or both the lines?

Reply:
just before `punkt` :)

Reply:
done :) @Makman2 now please merge it sir, i am starving to move on to another one 😄